### PR TITLE
ADM-953 [frontend][backend]: add timezone to request body and re-calculate and refactor

### PR DESCRIPTION
**Pull request by zhou-yinyuan**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Comment by guzhongren** [comment]
ref to `https://day.js.org/docs/en/installation/browser`

File: frontend/e2e/specs/side-path/unhappy-path.spec.ts [hunk 1]
```diff
@@ -3,12 +3,14 @@ import {
...
 } from '../../fixtures/import-file/unhappy-path-file';
 import { BOARD_METRICS_RESULT, DORA_METRICS_RESULT } from '../../fixtures/create-new/report-result';
+import adjustBrowserTimezone from '../../utils/set-chrome-default-timezone';
 import { test } from '../../fixtures/test-with-extend-fixtures';
 import { clearTempDir } from '../../utils/clear-temp-dir';
 import { format } from '../../utils/date-time';
 
 test.beforeAll(async () => {
   await clearTempDir();
+  await adjustBrowserTimezone();
```
[the remark this answers is not in the document]
Comment (zhou-yinyuan) [reply frontend/e2e/specs/side-path/unhappy-path.spec.ts]: have deleted

File: frontend/e2e/utils/set-chrome-default-timezone.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { chromium } from '@playwright/test';
+
+export default async function adjustBrowserTimezone() {
+  const browser = await chromium.launch();
+  await browser.newContext({
+    timezoneId: 'Asia/Shanghai', // 设置所需的时区
```
[the remark this answers is not in the document]
Comment (zhou-yinyuan) [reply frontend/e2e/utils/set-chrome-default-timezone.ts]: yes, we have deleted the code

File: backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java [hunk 3]
```diff
@@ -713,7 +714,8 @@ void shouldGetCardsWhenCallGetStoryPointsAndCycleTimeGivenStoryPointKeyFromEnvir
 		StoryPointsAndCycleTimeRequest storyPointsAndCycleTimeRequest = STORY_POINTS_FORM_ALL_DONE_CARD().build();
 		storyPointsAndCycleTimeRequest.setOverrideFields(jiraBoardSetting.getOverrideFields());
 		CardCollection cardCollection = jiraService.getStoryPointsAndCycleTimeAndReworkInfoForDoneCards(
-				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "");
+				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "",
+				ZoneId.of("Asia/Shanghai"));
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java:718]: Duplicated code

**Review by guzhongren — APPROVED** [review]

File: backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java (hunk 3 above)
Comment (zhou-yinyuan) [reply backend/src/test/java/heartbeat/service/jira/JiraServiceTest.java:718]: I don't understand, I don't find any duplicated code. if you say the ```jiraService.getStoryPointsAndCycleTimeAndReworkInfoForDoneCards(
				storyPointsAndCycleTimeRequest, jiraBoardSetting.getBoardColumns(), List.of("Zhang San"), "",
				ZoneId.of("Asia/Shanghai"))``` is duplicated, the params is different.

**Review by Mandy-Tang — APPROVED** [review]

**Pull request merged**